docs/kyma/docs/031-gs-local-installation.md

```diff
@@ -61,30 +64,9 @@ scripts/is-installed.sh
 Read the [Reinstall Kyma](025-details-local-reinstallation.md) document to learn how to reinstall Kyma without deleting the cluster from Minikube.
 To learn how to test Kyma, see the [Testing Kyma](026-details-testing.md) document.
 
-### Custom Resource file
+### Customize the Kyma installation
 
-The Custom Resource file contains controls the Kyma installer, which is a proprietary solution based on the [Kubernetes operator](https://coreos.com/operators/). The file contains the basic information that defines Kyma installation.
-Find the custom resource template [here](../../../installation/resources/installer-cr.yaml.tpl).
-
-### Control the installation process
-
-To trigger the installation process, set the **action** label to `install` in the metadata of the Custom Resource with the installer configuration.
-To trigger the deinstallation process, set the **action** label to `uninstall` in the metadata of the Custom Resource with the installer configuration.
-
-### Generate a new Custom Resource file
-
-Use the `create-cr.sh` script to generate the Custom Resource file. The script accepts these arguments:
-
-- `--output` - mandatory. The location of the Custom Resource output file
-- `--url` - the URL of the Kyma package to install
-- `--version` - the Kyma version
-- `--ipaddr` - the load balancer IP
-- `--domain` - the instance domain
-
-For example:
-```
-$ ./installation/scripts/create-cr.sh --output kyma-cr.yaml --url {Kyma_TAR.GZ_URL} --version 0.0.1 --domain kyma.local
-```
+Read the [Installer Custom Resource](./037-CRD.md) document to learn how to customize the Kyma installation by customizing the installer Custom Resource file.
```

Comment:
remember that for now links still do not work in the docs UI so please remove it